src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -223,7 +223,7 @@ const JobHistory = _.flow(
                     status, submissionEntity
                   } = filteredSubmissions[rowIndex]
                   return h(Fragment, [
-                    (collapsedStatuses(workflowStatuses).running && status !== 'Aborting') && buttonPrimary({
+                    (!isTerminal(status) && status !== 'Aborting') && buttonPrimary({
```

Comment:
I'm not sure this is the right definition of `isTerminal`, at least in this case.

```
const isTerminal = submissionStatus => submissionStatus === 'Aborted' || submissionStatus === 'Done'
```

I think it should include `Failed`... though I don't know if including `Failed` is appropriate for where this `isTerminal` is already being used.

Reply:
I believe "Failed" shows up as status "Done", too

Reply:
Oh right... one is for the entire submission and the other is for each job.